aws/resource_aws_glue_job_test.go

```diff
@@ -729,6 +730,7 @@ resource "aws_glue_job" "test" {
   command {
     name            = "pythonshell"
     script_location = "testscriptlocation"
+    python_version =  "3"
```

Comment:
Let's remove the version from the configuration to validate that setting no python_version works as expected. 
```suggestion

```